README.md

```diff
@@ -68,12 +68,11 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- Using one of our [executable installers](https://github.com/spyder-ide/spyder/releases)
```

Comment:
Please remove this line. I'm not going to produce Windows installers for Spyder 3.0.

The main reason is that we're now creating wheels for Spyder that pull all our dependencies correctly. The Windows installers can't install dependencies (much less PyQt) so they are kind of deceiving in that respect.